packages/flame/lib/src/game/flame_game.dart

```diff
@@ -86,6 +86,24 @@ class FlameGame extends ComponentTreeRoot
     setMounted();
   }
 
+  /// Setting to true will remove all children and process any remaining life
+  /// cycle events.
+  bool disposeAllOnRemove = false;
```

Comment:
We agreed that this should be removed right, and `onRemove` should be left empty?

Reply:
OOPS!  I forgot to remove that entirely.  `onRemove` was added by me to `FlameGame` so that whole section can be removed.